### ISSUE TITLE
(0.20.0) Android release version can not play the sound problem。

### ISSUE THREAD

**Issue by qiaoxiaogg** [luanpotter/audioplayers#1001]

**Comment by erickzanardo** [comment]
Please follow the issue template and add description of the problem.

**Comment by comESK** [comment]
I'm also having a similar issue on `0.20.0` (iOS is fine) with 
```dart
import 'package:audioplayers/audioplayers.dart';

final AudioPlayer audioPlayer = new AudioPlayer(mode: PlayerMode.LOW_LATENCY);
final AudioCache audioCache = new AudioCache(
  prefix: 'assets/audio/',
  fixedPlayer: audioPlayer,
  duckAudio: true,
  respectSilence: true,
);



/// In root initState
audioCache.loadAll(list_of_all_audio_assets);


/// Then, in a Button's onPressed:
audioCache.play('already_loaded_asset.wav');
```
and this is output:

```
E/flutter (29133): [ERROR:flutter/lib/ui/ui_dart_state.cc(209)] Unhandled Exception: PlatformException(Unexpected error!, No enum constant l.a.a.d.ReleaseMode.STOP, java.lang.IllegalArgumentException: No enum constant l.a.a.d.ReleaseMode.STOP
E/flutter (29133): 	at java.lang.Enum.valueOf(Enum.java:257)
E/flutter (29133): 	at l.a.a.d.valueOf(Unknown Source:2)
E/flutter (29133): 	at l.a.a.a.b(Unknown Source:210)
E/flutter (29133): 	at l.a.a.a.a(Unknown Source:10)
E/flutter (29133): 	at g.a.c.a.j$a.a(Unknown Source:17)
E/flutter (29133): 	at io.flutter.embedding.engine.f.b.a(Unknown Source:49)
E/flutter (29133): 	at io.flutter.embedding.engine.FlutterJNI.handlePlatformMessage(Unknown Source:4)
E/flutter (29133): 	at android.os.MessageQueue.nativePollOnce(Native Method)
E/flutter (29133): 	at android.os.MessageQueue.next(MessageQueue.java:325)
E/flutter (29133): 	at android.os.Looper.loop(Looper.java:142)
E/flutter (29133): 	at android.app.ActivityThread.main(ActivityThread.java:6944)
E/flutter (29133): 	at java.lang.reflect.Method.invoke(Native Method)
E/flutter (29133): 	at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:327)
E/flutter (29133): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1374)
E/flutter (29133): , null)
E/flutter (29133): #0      StandardMethodCodec.decodeEnvelope (package:flutter/src/services/message_codecs.dart:607)
E/flutter (29133): #1      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:156)
E/flutter (29133): <asynchronous suspension>
E/flutter (29133): #2      AudioPlayer.invokeMethod (package:audioplayers/src/audioplayer.dart:151)
E/flutter (29133): <asynchronous suspension>
E/flutter (29133): #3      AudioCache.play (package:audioplayers/src/audio_cache.dart:162)
E/flutter (29133): <asynchronous suspension>
```

My `flutter doctor`:
```
[✓] Flutter (Channel stable, 2.5.0, on macOS 11.5.2 20G95 darwin-arm, locale en-US)
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
[✓] Xcode - develop for iOS and macOS
[✓] Chrome - develop for the web
[✓] Android Studio (version 4.2)
[✓] Android Studio (version 2020.3)
[✓] VS Code (version 1.60.0)
[✓] Connected device (3 available)
```

Specific Android specs that I reproduced the issue on:
Galaxy S7 (SM G930V) Android 8.0.0 (API 26)

Let me know if that helps at all while you wait for OP!

**Comment by martin-73** [comment]
The same is here.

**Comment by luanpotter** [comment]
It's an error of how we handle enums on android, it seems on release mode the packages get all mangled up: `l.a.a.d.ReleaseMode`. I will fix it shortly!

**Comment by luanpotter** [comment]
I only tested it on debug mode, sorry

**Comment by luanpotter** [comment]
https://github.com/luanpotter/audioplayers/pull/1002

**Comment by luanpotter** [comment]
Fixed on `0.20.1`
